src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/playback/GerritMissedEventsPlaybackManager.java

```diff
@@ -92,7 +93,8 @@
     /**
      * Server Timestamp.
      */
-    protected EventTimeSlice serverTimestamp = null;
+    protected static EventTimeSlice serverTimestamp = null;
```

Comment:
This will make it not a **server**Timestamp but a global timestamp. Seems possible to mess things up making it static since there is one manager per server.

Reply:
It's no longer static now.